### PR TITLE
[RHOAIENG-2989] artifact node drawer details section

### PR DESCRIPTION
**Pull request by jpuzz0**
Starts: [RHOAIENG-2989](https://issues.redhat.com/browse/RHOAIENG-2989)

## Description
Fetches the artifact context for the run, now displaying those MLMD artifact details. Virtualization will be a follow-up PR.

image

image
(cc yannnz)

## Test Impact
Will include tests in follow-up PR if applicable.

## Request review criteria:

Self checklist (all need to be checked):
- [x] The developer has manually tested the changes and verified that the changes work
- [x] Commits have been squashed into descriptive, self-contained units of work (e.g. 'WIP' and 'Implements feedback' style messages have been removed)
- [ ] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
- [ ] The developer has added tests or explained why testing cannot be added (unit or cypress tests for related changes)

If you have UI changes: 
- [x] Included any necessary screenshots or gifs if it was a UI change.
- [x] Included tags to the UX team if it was a UI/UX change (find relevant UX in the [SMEs](https://github.com/opendatahub-io/odh-dashboard/tree/main/docs/smes.md) section).

After the PR is posted & before it merges:
- [ ] The developer has tested their solution on a cluster by using the image produced by the PR to `main`

### CODE REVIEW

File: frontend/src/pages/pipelines/global/experiments/artifacts/useGetArtifactsList.ts
Comment (DaoDaoNoCode) [inline frontend/src/pages/pipelines/global/experiments/artifacts/useGetArtifactsList.ts:1]: Why make type changes in this file from null to undefined?

**Review by DaoDaoNoCode — APPROVED** [review]
Code looks good, just left a small question that doesn't block the approval.
/lgtm

File: frontend/src/pages/pipelines/global/experiments/artifacts/useGetArtifactsList.ts
Comment (jpuzz0) [reply frontend/src/pages/pipelines/global/experiments/artifacts/useGetArtifactsList.ts:1]: There was situation where the type could have been null | undefined, and we weren't using any null specific checks so this simplified things.

**Comment by Gkrumbach07** [comment]
/approve

lets make sure not to close out the issue as we still need to add metrics viz

**Pull request merged**